### ISSUE TITLE
Prompt to upgrade doesn’t mention that the command has to be run twice

### ISSUE THREAD

**Issue by macgills** [gradle/gradle#31802]
### Issue type

Wrong or misleading information

### Problem description

In the release notes it just says to run 

    ./gradlew wrapper --gradle-version=XXXX

but that doesn’t actually fully update the wrapper and only after you run it a 2nd time will you actually have that updated wrapper. https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:upgrading_wrapper

Ideally just running that command would update your version+wrapper but if that isn’t possible then it should be explicitly called out everywhere the upgrade wrapper command is mentioned that unless you do it twice then you have probably done it wrong. 

I don’t think it is reasonable for the wrapper command to not update the wrapper and for this information to be relatively hidden. 

### Page with the problem

https://docs.gradle.org/8.10.1/release-notes.html

**Comment by jbartok** [comment]
This issue will be closed as a duplicate of

* https://github.com/gradle/gradle/issues/29330

Please add your use case and 👍 to that issue.

If you think our analysis is wrong, please provide us with more detailed information explaining why.

**Comment by macgills** [comment]
Apologies for not seeing the duplicate, though I would rather it be a duplicate of #884 as the fix of the root cause would be preferred. Is there an actual path forward that is not simply a documentation fix? With a major version increment in the near future surely now would be a good time for a breaking change or deprecation and replacement task?
